doc/config.rst

```diff
@@ -1663,6 +1663,16 @@ These options influence LaTeX output. See further :doc:`latex`.
      ``'maketitle'``
         "maketitle" call, default ``'\\maketitle'``.  Override if you want to
         generate a differently-styled title page.
+     ``'fvset'``
```

Comment:
The settings is like a contract to users.
The documentation has a very long life. So we should keep a compatibility of the new settings on feature sphinx.
I think we have to be careful to add them.

We already have `preamble` key to customize these trivial settings.
So I think no special key is not needed here.